internal/connector/internal/handlers/processor_secrets.go

```diff
@@ -0,0 +1,36 @@
+package handlers
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/internal/api/dbapi"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/internal/services/vault"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+const OwningProcessorResourcePrefix = "/v1/processor/"
```

Comment:
@manstis Should this be `v2Alpha1` to align with the path under which we are exposing the Processor API?

Reply:
Yes it should. Good spot! I will change.